### ISSUE TITLE
Bios Version

### ISSUE THREAD

**Issue by tuxymac** [echokk11/AMD-3900x_MSI-MEG-UNIFY-X570_5700xt_hackintosh_opencore#1]
Which Bios version are you using?
It seems that it works only with a specific Bios version.

Thanks

**Comment by echokk11** [comment]
I use the eariest version I could download in MSI website.
```
Version
7C35vA0
Release Date
2019-09-24
File Size
14.48 MB
```

https://www.msi.com/Motherboard/support/MEG-X570-UNIFY#down-bios

**Comment by YahuiWong** [comment]
@tuxymac Which Bios version are you using? Maybe we can avoid mistakes

**Comment by tuxymac** [comment]
I use E7c35AMS.a60 09/07/2020
I guess I need to downgrade in order to get Monterey working

**Comment by tuxymac** [comment]
Do you have BT and Wi-Fi working on your rig?

**Comment by echokk11** [comment]
Yes, I use a PCIE card (FV T919) instead of the AX200, and disabled the AX200 BT in `USBPorts.kext`
